pkg/providers/snow/apibuilder.go

```diff
@@ -235,3 +242,24 @@ func SnowMachineTemplate(name string, machineConfig *v1alpha1.SnowMachineConfig)
 		},
 	}
 }
+
+func bottlerocketBootstrapSnow(image releasev1.Image) bootstrapv1.BottlerocketBootstrapContainer {
```

Comment:
Why is this here and not in the clusterapi package? Do we want a bootstrap file for this instead?

Reply:
bc it's a snow specific bootstrap container including the settings like name and mode. I can move it to another file